src/cplscheme/BiCouplingScheme.cpp

```diff
@@ -138,23 +138,67 @@ CouplingData *BiCouplingScheme::getReceiveData(
 void BiCouplingScheme::exchangeInitialData()
 {
   // F: send, receive, S: receive, send
+  bool recvInitialData = true;
+  bool sendInitialData = true;
```

Comment:
Naming here is really unfortunate as these variable do not store whether initial data is send or received.
Wouldn't it be easier if the variable of `sendData` and `receiveData` had the same name, e.g. `initialCommunication` or `initialize`?

Reply:
Sure. The name is only temporary and will vanish again in #1414, but I don't really care.

Reply:
Done. c58b6128a522af256756516fbfc81b8613a52fe4